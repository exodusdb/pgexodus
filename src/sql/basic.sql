set datestyle = "ISO";
set intervalstyle = "postgres";
select
 exodus.extract_text(E'a1 \x1E b1 \x1D b2a \x1C b2b \x1E cc',2,2,2),
 exodus.extract_date('20000',0,0,0),
 exodus.extract_time('86399',1,0,0),
 exodus.extract_datetime('20000.86399',1,1,0),
 exodus.extract_number('123.456', 1, 0, 0),
 exodus.count('ABBA','A');
