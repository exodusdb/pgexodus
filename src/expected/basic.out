set datestyle = "ISO";
set intervalstyle = "postgres";
select
 exodus.extract_text(E'a1 \x1E b1 \x1D b2a \x1C b2b \x1E cc',2,2,2),
 exodus.extract_date('20000',0,0,0),
 exodus.extract_time('86399',1,0,0),
 exodus.extract_datetime('20000.86399',1,1,0),
 exodus.extract_number('123.456', 1, 0, 0),
 exodus.count('ABBA','A');
 extract_text | extract_date | extract_time |  extract_datetime   | extract_number | count 
--------------+--------------+--------------+---------------------+----------------+-------
  b2b         | 2022-10-03   | 23:59:59     | 2022-10-03 00:00:00 |        123.456 |     2
(1 row)

